### ISSUE TITLE
Installing on M3 Mac Pro

### ISSUE THREAD

**Issue by jrad77** [chauff/paper-note-filler#5]
To get this to install properly, I needed to 

1. install npm: `brew install npm`
2. clone this whole repo in the `.plugin` directory

then

```
cd paper-note-filler
npm install
npm run dev
```

Then once I loaded Obsidian, I was able to open the plugin correctly.

Before doing this, and following the steps in the README resulted in an error toast saying that it "Failed to load plugin". 

Obsidian debug info:

> SYSTEM INFO:
> 	Obsidian version: v1.5.12
> 	Installer version: v1.5.11
> 	Operating system: Darwin Kernel Version 23.4.0: Fri Mar 15 00:12:37 PDT 2024; root:xnu-10063.101.17~1/RELEASE_ARM64_T6031 23.4.0
> 	Login status: logged in
> 	Catalyst license: none
> 	Insider build toggle: off
> 	Live preview: on
> 	Base theme: adapt to system
> 	Community theme: none
> 	Snippets enabled: 0
> 	Restricted mode: off
> 	Plugins installed: 6
> 	Plugins enabled: 5
> 		1: Smart Connections v2.1.50
> 		2: ReadItLater v0.4.0
> 		3: Natural Language Dates v0.6.2
> 		4: Copilot v2.5.2
> 		5: Paper Note Filling v1.0.1
> 
> RECOMMENDATIONS:
> 	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

**Comment by asapsmc** [comment]
Ran into the same issues here in MacOS M1. Tried the same solution, but got stuck when I do : 
```
npm install
npm warn old lockfile
npm warn old lockfile The package-lock.json file was created with an old version of npm,
npm warn old lockfile so supplemental metadata must be fetched from the registry.
npm warn old lockfile
npm warn old lockfile This is a one-time fix-up, please be patient...
npm warn old lockfile
```
and this never goes away. I already tried `npm install --force` and `npm install --package-lock-only`, but nothing solves this. 

Then I changed node version to 16.20.2(a version closer to 16.11.6, as identified in package.json) but now this process (npm install) is able to start but stalls close to 100%. 

@jrad77 any idea how might I solve this? I'm with Obsidian 1.6.5., npm 10.8.1 (originally, but then changed to node 16,20,2, using npm 8.19.4).
